packages/edit-site/src/store/selectors.js

```diff
@@ -136,7 +125,7 @@ export function getTemplateType( state ) {
  * @return {Object} Page.
  */
 export function getPage( state ) {
```

Comment:
This is very unclear, as far as I know it's used to set the "context" of the template (to fill it with actual content from a given post), I think it needs to be clarified in the code as well. It doesn't really return a "page" so the name is confusing. This probably deserves a refactoring on its own.

Reply:
I agree, maybe the name could be getPageContext (?).